sound/soc/bcm/Kconfig

```diff
@@ -85,3 +85,12 @@ config SND_BCM2708_SOC_ADAU1977_ADC
 	select SND_SOC_ADAU1977_I2C
 	help
 	  Say Y or M if you want to add support for ADAU1977 ADC.
+
+config SND_BCM2708_SOC_FE_PI_AUDIO
+	tristate "Support for Fe-Pi-Audio"
+	depends on SND_BCM2708_SOC_I2S || SND_BCM2835_SOC_I2S
+	select SND_SOC_SGTL5000
+	select REGULATOR
```

Comment:
The SGTL5000 codec driver needs regulator. I tried without it, did not work. Got the following without regulator....

[   14.331875] sgtl5000 1-000a: sgtl5000 revision 0x11
[   14.414956] sgtl5000 1-000a: this setup needs regulator support in the kernel
[   14.414991] sgtl5000 1-000a: Failed to register vddd internal supplies: -22
[   14.415004] sgtl5000 1-000a: ASoC: failed to probe component -22
[   14.415061] snd-fe-pi-audio soc:sound: ASoC: failed to instantiate card -22
[   14.415156] snd-fe-pi-audio soc:sound: snd_soc_register_card() failed: -22
[   14.415193] snd-fe-pi-audio: probe of soc:sound failed with error -22